core/src/apps/cardano/layout.py

```diff
@@ -54,6 +54,11 @@
     CardanoCertificateType.STAKE_POOL_REGISTRATION: "Stakepool registration",
 }
 
+WARNING_TITLE_CHANGE_OUTPUT_PATH = "Change output path"
```

Comment:
Given that we aren't doing it anywhere else (AFAIK) I wouldn't introduce those warning title constants, especially if the given string is used once anyway

Reply:
True. I tried various structuring of those functions so the constants come from there. They aren't really needed in this final form.